levels/lovelace_tower/objectives/api-03-fetch/walkthrough.md

```diff
@@ -0,0 +1,63 @@
+
+## Using fetch
+
+The `fetch` function takes two arguments, one is the URL that you're sending the request to and the other is the configuration. We'll be focussing on just a few of the configuration options here (the `method` option being among them), but there are [tons of others](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters) to play around with. The `method` option is one of the most important to set when making requests, because it often drastically changes the response you get. You can set `method` to any of these [HTTP Verbs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), more on that coming up.
+
+## What are HTTP Verbs?
+
+HTTP Verbs (i.e. methods) represent the types of requests you can make. The most commonly used methods are `GET`, `POST`, `PATCH`, `PUT`, and `DELETE`.
+
+- `GET` asks for some sort of data or information from the other device
+- `POST` asks for something to be created by the other device
+- `PUT` asks for something that exists already to be replaced entirely by the other device with what's being sent in the request
+- `PATCH` asks for something that exists already to be updated by the other device using a set of instructions being sent in the request
+- `DELETE` asks for something to be deleted by the other device
+
+Some methods require additional data to be sent along with the request, while others do not. Try experimenting with them to get the hang of things! Now let's take a look at a couple of `fetch` examples:
+
+```js
+const response = await fetch("some_url", {
+  method: "GET",
+});
+
+console.log(response);
+```
+
+Notice the `await` keyword here. The `fetch` function is asynchronous and must be awaited or passed a callback via promise chaining. The example is making a fetch request to a fake URL, but any valid URL would work! Also, pay attention to how we're configuring the request -- the second argument we're passing in is an object and our request uses [the properties](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters:~:text=A%20Request%20object.-,options,-Optional) of that object as the different parts of it's configuration.
+
+```js
+const response = await fetch("some_url", {
+  method: "POST",
+  body: "Hello World",
+});
+
+console.log(response);
+```
+
+This example is similar to the last, only this time we're making a `POST` request with another option in our configuration object. This type of request is one that needs data to be sent along with it, and we accomplish that with the `body` property. The example is requesting that another computer create something new using the `Hello World` data we supplied.
```

Comment:
Similar comment to above, that POST being for creating something new is probably fairly REST specific language